config/jobs/kubernetes-security/generated-security-jobs.yaml

```diff
@@ -2341,7 +2341,10 @@ presubmits:
         - --timeout=120m
         image: gcr.io/k8s-testimages/kubekins-e2e:v20190116-8e0f20924-master
         name: ""
-        resources: {}
+        resources:
```

Comment:
note to self: when knative-build &/or pipeline support is done we should look into splitting the build into a step with higher requests. cc @fejta is that possible?